crates/dbs-virtio-devices/CHANGELOG.md

```diff
@@ -1,13 +1,22 @@
 # CHANGELOG
 
+## v0.1.0
+
+### Added 
+
+- This is the initial release for dbs-virtio-devices
+
```

Comment:
The New releases are usually on top?

Reply:
In dbs crates, we put new releases in the bottom..